tests/by-util/test_hashsum.rs

```diff
@@ -338,6 +365,29 @@ fn test_check_md5sum() {
     }
 }
 
+#[test]
+fn test_check_md5sum_not_enough_space() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    for f in &["a", " b"] {
```

Comment:
Is it intentional that the second string starts with a space?
```suggestion
    for f in ["a", "b"] {
```